modules/dns/versions.tf

```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = ">= 1.49.0"
     }
+    local = {
+      source  = "hashicorp/local"
+      version = ">= 2.4.0"
+    }
```

Comment:
why do we need this?

Reply:
We need this provider for the local_file resource block as suggested in the [registry](https://registry.terraform.io/providers/hashicorp/local/latest/docs/resources/file).